config/config.exs

```diff
@@ -19,6 +19,13 @@ config :logger, :console,
   metadata: [:request_id, :room_id]
 
 config :phoenix, :json_library, Jason
+config :phoenix, :logger, false
+
+config :logger,
+  compile_time_purge_matching: [
+    [application: :membrane_rtc_engine, level_lower_than: :warning],
+    [application: :membrane_rtc_engine_webrtc, level_lower_than: :warning]
```

Comment:
Why only webrtc is included? What about hls and rtsp?

Reply:
I first thought there were not many logs from HLS and RTSP, so it isn't necessary to remove them, but maybe that is the right direction to be consistent.